src/bin/nydusd/main.rs

```diff
@@ -504,8 +504,8 @@ fn process_fs_service(
             .value_of("failover-policy")
             .unwrap_or(&"flush".to_string())
             .try_into()
-            .inspect(|_e| {
-                error!("Invalid failover policy");
+            .inspect_err(|e| {
```

Comment:
Thanks, please rebase the latest master, it seems have a merge op introduces the additional commit.